demos/common/python/models/yolo.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, param, sides):
     def __init__(self, ie, model_path, labels=None, keep_aspect_ratio=False, threshold=0.5, iou_threshold=0.5):
         super().__init__(ie, model_path)
 
+        self.is_tiny = self.net.name.lower().find('tiny') != -1  # Weak way to distinguish between YOLOv4 and YOLOv4-tiny
```

Comment:
what about using number of outputs to define full or tiny model version is loaded?
naming is not looks less robust heuristic

Reply:
That's the problem. MO produces 5 outputs (3 of them are fake).

Reply:
it is expected behaviour or bug in MO?
(for me looks like a bug). 
Do you not try to explicitly specify outputs in MO?

Reply:
This is not appropriate way to determine model subtype. More elegant way is to parse output layers and ignore all outputs with `Fake` in name. Still, it is weak and vulnarable way.

Reply:
@eaidova, yes, I've tried it by `--output` flag but it's not extra outputs from origin model but outputs which are produced by MO ops.

Reply:
it sound sad, but ok